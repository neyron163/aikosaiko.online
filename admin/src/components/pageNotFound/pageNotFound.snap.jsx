import React from 'react';
import renderer from 'react-test-renderer';

import {PageNotFound} from './pageNotFound';

describe('<PageNotFound />', () => {
    it('should render correctly', () => {
        const tree = renderer.create(<PageNotFound />);
        expect(tree.toJSON()).toMatchSnapshot();
    });
});
