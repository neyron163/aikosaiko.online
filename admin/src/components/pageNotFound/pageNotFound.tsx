import React from 'react';

/**
 * PageNotFound
 */
export const PageNotFound = () => <div>404 - page not found</div>;
