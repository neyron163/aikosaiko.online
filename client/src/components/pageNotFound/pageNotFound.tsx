import React from 'react';

export const PageNotFound = () => (<div>404 - page not found</div>)