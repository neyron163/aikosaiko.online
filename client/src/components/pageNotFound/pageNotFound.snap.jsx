import React from 'react';
import renderer from 'react-test-renderer';

import {PageNotFound} from './pageNotFound';

let props;

beforeEach(() => {
    props = {
        location: {
            pathname: '/',
        },
    };
});

describe('<PageNotFound />', () => {
    it('should render correctly', () => {
        const tree = renderer.create(<PageNotFound {...props} />);
        expect(tree.toJSON()).toMatchSnapshot();
    });

    it('should render correctly if pathname = admin', () => {
        props.location.pathname = '/admin';
        const tree = renderer.create(<PageNotFound {...props} />);
        expect(tree.toJSON()).toMatchSnapshot();
    });
});
