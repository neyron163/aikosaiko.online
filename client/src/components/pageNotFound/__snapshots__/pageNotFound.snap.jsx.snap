// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> should render correctly 1`] = `
<div>
  404 - page not found
</div>
`;

exports[`<PageNotFound /> should render correctly if pathname = admin 1`] = `null`;
